acceptance/testdata/analyzer/analyze-image/container/cnb/bad-stack.toml

```diff
@@ -0,0 +1 @@
+[run-images
```

Comment:
Are we using this file?

Reply:
acceptance/analyzer_test.go:1105

Reply:
@yaelharel thanks! @jabrown85 could we also move this to the Dockerfile?

Reply:
11a5a76